src/Entities/Builders/UserBuilder.php

```diff
@@ -18,7 +18,7 @@ public static function buildFromResponse(array $userData): User
 
         $user->setId($userData["id"]);
         $user->setEmail($userData["user_attributes"]["email"]);
-        $user->setEmailVerified($userData["user_attributes"]["email_verified"]);
+        $user->setEmailVerified($userData["user_attributes"]["email_verified"] ?? '');
```

Comment:
```suggestion
        $user->setEmailVerified($userData["user_attributes"]["email_verified"] ?? "");
```

Please be consistent with our codestyle.

Reply:
updated